apps/web/src/components/healthcare/ClinicalWorkflowPatterns.tsx

```
@@ -0,0 +1,851 @@
+
+  return (
+    <NeumorphicCard
+      variant={hasUrgentStatus ? 'alert' : 'patient'}
+      className={cn(
+        'cursor-pointer transition-all duration-200 hover:scale-[1.02]',
+        hasUrgentStatus && 'ring-2 ring-red-500 ring-opacity-50',
+        compact ? 'p-4' : 'p-6',
+        className
+      )}
+      onClick={() => onSelectPatient?.(patient)}
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)